src/SlidingSyncManager.ts

```diff
@@ -64,7 +65,11 @@ const DEFAULT_ROOM_SUBSCRIPTION_INFO = {
     ],
 };
 
-export type PartialSlidingSyncRequest ={ filters?: MSC3575Filter, sort?: string[], ranges?: number[][] };
+export type PartialSlidingSyncRequest = {
+    filters?: MSC3575Filter;
+    sort?: string[];
+    ranges?: number[][];
```

Comment:
Isn't one of these a pair rather than a full array? e.g. `[number, number][]`

Reply:
The pair elements can also be given names e.g. `[startIndex: number, endIndex: number][]`